<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Stream</class>
 <widget class="PlayerPluginInterface" name="GUI_Stream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>120</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="Gui::ComboBox" name="combo_stream">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="MenuToolButton" name="btn_tool">
       <property name="text">
        <string notr="true">Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_play">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_listen">
       <property name="text">
        <string notr="true">Listen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="le_url">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::ComboBox</class>
   <extends>QComboBox</extends>
   <header>GUI/Utils/Widgets/ComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>MenuToolButton</class>
   <extends>QPushButton</extends>
   <header location="global">GUI/Utils/MenuTool/MenuTool.h</header>
  </customwidget>
  <customwidget>
   <class>PlayerPluginInterface</class>
   <extends>QWidget</extends>
   <header>Interfaces/PlayerPlugin/PlayerPlugin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
